pywbem/_recorder.py

```diff
@@ -320,10 +322,18 @@ def open_file(filename, file_mode='w'):
             existing file.  if 'a' is used, the data is appended to any
             existing file.
 
+        Returns:
+
+          File-like object.
```

Comment:
What is  a file-like object

Reply:
"File-like object" is a Python term for an object that behaves like a file (e.g. read, write, close). Here is one article about it, but not 100% exhaustive, I guess: https://stackoverflow.com/questions/4359495/what-is-exactly-a-file-like-object-in-python